src/main/java/com/redhat/darcy/webdriver/internal/TargetedWebElement.java

```diff
@@ -228,4 +230,9 @@ private WebElement targetedWebElement(WebElement element) {
                 .map(e -> new TargetedWebElement(locator, target, e))
                 .collect(Collectors.toList());
     }
+
+    @Override
+    public <X> X getScreenshotAs(OutputType<X> outputType) throws WebDriverException {
+        return null;
+    }
```

Comment:
Please move above with public, overridden API, and also implement instead of just returning null.